float32 err_x
float32 err_theta
